client/tailwind.config.ts

```diff
@@ -23,6 +23,7 @@ const config = {
         sans: ["var(--font-sans)", ...fontFamily.sans],
       },
       colors: {
+        customPrimary: "hsl(var(--customPrimary))",
```

Comment:
you could use `penni-main:"#0051CA"` here to make it more semantic. 
good job tho

Reply:
I found variables like "`--customPrimary`" should be included in `global.css`, I added this var in `global.css` like the 16th line: `--customPrimary: 225, 100%, 40%; /*#0051CA Penni Blue*/` 👀

Reply:
add `"penni-main“:"#0051CA"` here and use it like `className="bg-penni-main"`
we use tailwind css so, from my perspective, we should configure the theme color in `tailwind.config.ts`

Reply:
I'll modify it